src/@ionic-native/plugins/app-version/index.ts

```diff
@@ -37,28 +37,28 @@ import { Cordova, Plugin, IonicNativePlugin } from '@ionic-native/core';
 export class AppVersion extends IonicNativePlugin {
 
   /**
-   * Returns the name of the app
+   * Returns the name of the app, e.g.: "My Awesome App"
    * @returns {Promise<string>}
    */
   @Cordova()
   getAppName(): Promise<string> { return; }
 
   /**
-   * Returns the package name of the app
+   * Returns the package name of the app, e.g.: "com.example.myawesomeapp"
    * @returns {Promise<string>}
    */
   @Cordova()
   getPackageName(): Promise<string> { return; }
 
   /**
-   * Returns the build identifier of the app
-   * @returns {Promise<string>}
+   * Returns the build identifier of the app, e.g.: 10203
+   * @returns {Promise<number>}
    */
   @Cordova()
-  getVersionCode(): Promise<string> { return; }
+  getVersionCode(): Promise<number> { return; }
```

Comment:
That's true for Android but not for iOS.
An iOS version code can look like this "1.2.3.4"

Reply:
Thanks for the review. You are right.

I just checked the code: https://github.com/whiteoctober/cordova-plugin-app-version/blob/master/src/ios/AppVersion.m

It seems it is returning the `CFBundleVersion`, then I found here in the docs that is seems to be the build version, as you say, as a string: https://developer.apple.com/library/archive/technotes/tn2420/_index.html

I just updated it and the doc string accordingly.